analysis/deep2_r23_o32/run_general_script.py

```diff
@@ -1,4 +1,23 @@
 #!/usr/bin/env python3
 from Zcalbase_gal.analysis.deep2_r23_o32 import general
+from Metallicity_Stack_Commons import get_user
 dataset = 'n_Bins'
 general.run_grid_r23_o32_analysis(dataset)
+
+fitspath_ini = get_user()
+general.run_grid_plots(fitspath_ini, dataset, raw=False, apply_dust=False,
+                       revised=False, individual=False)
+general.run_grid_plots(fitspath_ini, dataset, raw=False, apply_dust=False,
+                       revised=True, individual=False)
+general.run_grid_plots(fitspath_ini, dataset, raw=True, apply_dust=False,
+                       revised=False, individual=False)
+general.run_grid_plots(fitspath_ini, dataset, raw=True, apply_dust=False,
+                       revised=True, individual=False)
+general.run_grid_plots(fitspath_ini, dataset, raw=False, apply_dust=True,
+                       revised=False, individual=False)
+general.run_grid_plots(fitspath_ini, dataset, raw=False, apply_dust=True,
+                       revised=True, individual=False)
+general.run_grid_plots(fitspath_ini, dataset, raw=True, apply_dust=True,
+                       revised=False, individual=False)
+general.run_grid_plots(fitspath_ini, dataset, raw=True, apply_dust=True,
+                       revised=True, individual=False)
```

Comment:
You can simplify this using a `dict` approach. Take a look at [this](https://github.com/astrochun/Metallicity_Stack_Commons/blob/master/tests/test_error_prop.py#L10-L38)

Reply:
fixed